Essentials/src/com/earth2me/essentials/commands/Commandhat.java

```diff
@@ -35,6 +35,10 @@ protected void run(final Server server, final User user, final String commandLab
         } else {
             final ItemStack hand = user.getItemInHand();
             if (hand != null && hand.getType() != Material.AIR) {
+                if (user.isAuthorized("essentials.hat.exempt." + hand.getType().name())) {
```

Comment:
You should `toLowerCase` the material name.

Reply:
I had considered that at first but I didn't see that in the original issue. Just pushed a change to handle that.